src/shared/utils/utils.ts

```diff
@@ -122,23 +122,15 @@ export const getTemplateURL = (document: any): string | undefined => {
   }
 };
 
-export const getTemplateData = <T extends WrappedDocument<OpenAttestationDocument> | OpenAttestationDocument>(
-  document: T
-):
-  | {
-      [key: string]: any;
-    }[]
-  | {
-      [key: string]: any;
-    } => {
-  if (isWrappedV3Document(document) || isRawV3Document(document)) {
-    return document.credentialSubject;
+export const getDocumentData = (document: WrappedDocument<OpenAttestationDocument>): OpenAttestationDocument => {
+  if (isWrappedV3Document(document)) {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { proof, ...rest } = document;
```

Comment:
try to rename the variable with _: `proof: _whatever`, should fix eslint I think

Reply:
it doesn't fix the eslint, i will use lodash omit